packages/canon/src/components/Select/Select.tsx

```diff
@@ -30,103 +40,72 @@ export const Select = forwardRef<HTMLDivElement, SelectProps>((props, ref) => {
     description,
     options,
     placeholder = 'Select an option',
-    size = 'medium',
-    required,
-    error,
+    size = 'small',
+    icon,
+    'aria-label': ariaLabel,
+    'aria-labelledby': ariaLabelledBy,
+    isRequired,
```

Comment:
Was this switched to match React Area better? Otherwise I prefer omitting bool prefixes and just having it be `required` like before